extensions/oidc-client/runtime/src/main/java/io/quarkus/oidc/token/propagation/AccessToken.java

```diff
@@ -17,4 +17,44 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 public @interface AccessToken {
+
+    /**
+     * Name of the configured OidcClient. Default OidcClient will be used when no value is specified.
+     */
+    String value() default "";
+
+    /**
+     * Exchange the current token with OpenId Connect Provider for a new token using either
```

Comment:
Hi @michalvavrik 

So, if we do not specify the client name here, then the token propagation properties apply (exchange token required property only)...
I wonder if just having a client name is enough. I should've done it in the properties too, i.e, I have 2 duplicating properties there, `exchangeToken` and the `client` name - but the client name is only relevant if the `exchangeToken` is enabled.

So may be here, we only have an `exchangeTokenClient` property, if it it set (`Default`, or any other named client name value) we enabled the token exchange, if it is not set we don't...

I feel it can help us avoid adding all the exchange attributes just to regulate or disable the token exchange. 
For example, you have a `DISABLE` enum, but then setting the client name makes no sense as well...

So, I propose to have an `exchangeTokenClient` property with no default value, please consider it

Reply:
Hey @sberyozkin ,

implementation wise I have no problem to do that. I think that `@AccessToken(exchangeTokenClient = "Default")` already does exactly same thing as the `@AccessToken("Default")`.IMO your proposal is not simplifying it, but I don't mind.

> For example, you have a DISABLE enum, but then setting the client name makes no sense as well...

but you can also disable token exchange for default client while it is enabled for all the others by configuration properties. While your proposal means you can't override configuration properties.

> So, I propose to have an exchangeTokenClient property with no default value, please consider it

No default value means the `exchangeTokenClient` would be required. You mean that default value will be an empty string I assume. So "" -> token exchange enabled/disabled depending on the configuration properties, while "Default" or any other value means enabled. np.

Reply:
Let's settle on your suggestion as I want to make change now.